src/components/pages/gallery/PreviewCard.tsx

```diff
@@ -158,6 +162,7 @@ export default function PreviewCard(props: IProps) {
         return () => {
             // cool cool cool
             isMounted.current = false;
+            cancellerRef.current && cancellerRef.current.exec();
```

Comment:
this does the job but need to clear the url from LRU cache and any other place its stored